sdk/storage/storage-file-share/CHANGELOG.md

```diff
@@ -171,7 +170,15 @@
   });
   ```
 
-## 12.0.0-preview.3 (2019.08)
+## 10.3.0 (2019-09)
```

Comment:
Seems date has been added to other change logs. Do you also need to change this one?

Reply:
I don't see that on the packages npm page. So I don't know what to add there. But its sufficient as it is for the sorting that happens.